release/models/types/openconfig-yang-types.yang

```diff
@@ -136,10 +141,7 @@ module openconfig-yang-types {
       "A 64-bit counter. A counter value is a monotonically increasing
       value which is used to express a count of a number of
       occurrences of a particular event or entity. When a counter64
-      reaches its maximum value, 2^64-1, it loops to zero.
-      Discontinuities in a counter are generally triggered only when
-      the counter is reset to zero, through operator or system
-      intervention.";
+      reaches its maximum value, 2^64-1, it loops to zero.";
```

Comment:
Curious why we don't just use consistent wording in both counter variants?

```
When the counter reaches its maximum value (2^32-1), it wraps to zero
When the counter reaches its maximum value (2^64-1), it wraps to zero
```

The IETF type variants are:

```
The counter32 type represents a non-negative integer
that monotonically increases until it reaches a
maximum value of 2^32-1 (4294967295 decimal), when it
wraps around and starts increasing again from zero.
```

```
The counter64 type represents a non-negative integer
that monotonically increases until it reaches a
maximum value of 2^64-1 (18446744073709551615 decimal),
when it wraps around and starts increasing again from zero.
```